website/scripts/check-playground-samples-js.ts

```diff
@@ -0,0 +1,32 @@
+import { spawnSync } from "child_process";
+import { globSync } from "glob";
+import { exit } from "process";
+
+let someFileError = false;
+const files = globSync("src/website/data/playground-samples/*/*/*.js");
+for (const file of files) {
+	const command = `yarn tsc --noEmit --allowJs --checkJs --skipLibCheck ../out/monaco-editor/monaco.d.ts ${file}`;
```

Comment:
If there is a better way to type check, please tell me.

Reply:
I think it is good, but you don't need the "command" variable. I don't know why ESLint didn't catch this.

Reply:
I think it is good, but you don't need the "command" variable.